torch/_dynamo/variables/functions.py

```diff
@@ -310,6 +310,11 @@ def num_parameters(self):
         return super().num_parameters() - 1
 
 
+class PartialInvocationFunctionVariable(UserMethodVariable):
+    def self_args(self):
+        return []
```

Comment:
For my information, how come we need a separate subclass for this?

Reply:
Don't need. Mentioned in DM. Will rm.